devices/xiaomi.js

```diff
@@ -1381,6 +1381,24 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.curtain.acn003'],
+        model: 'ZNCLBL01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara curtain driver E1',
+        fromZigbee: [fz.xiaomi_curtain_position, fz.battery, fz.cover_position_tilt, fz.aqara_opple],
```

Comment:
It seems to me that you need to replace `fz.cover_position_tilt` with `fz.xiaomi_curtain_position_tilt` as a more suitable one, and also remove `fz.xiaomi_curtain_position`, because it processes messages from the _genAnalogOutput_ cluster that I didn't see in the log (or are they reported but not in the given log?).